spec/factories/drivers.rb

```diff
@@ -0,0 +1,18 @@
+FactoryBot.define do
+  factory :driver do
+    name { FFaker::Name.name }
+    license_number {  }
```

Comment:
can we put something inside here?

Reply:
am i able to use ffaker for license_number?